Subject: Remove dev environment
From: Roman Gruber <roman.gruber@inf.unibe.ch>
Index: glados/config/web.php
===================================================================
--- glados.orig/config/web.php
+++ glados/config/web.php
@@ -129,21 +129,4 @@ $config = [
     'runtimePath' => '/var/lib/glados/runtime',
 ];
 
-if (YII_ENV_DEV) {
-    // configuration adjustments for 'dev' environment
-    if($_SERVER['PATH_INFO'] != '/event/stream') {
-        $config['bootstrap'][] = 'debug';
-        $config['modules']['debug'] = [
-            'class' => 'yii\debug\Module',
-            'allowedIPs' => ['10.16.0.222', '192.168.0.115', '127.0.0.1']
-        ];
-    }
-
-    $config['bootstrap'][] = 'gii';
-    $config['modules']['gii'] = [
-        'class' => 'yii\gii\Module',
-        'allowedIPs' => ['10.16.0.222', '192.168.0.115', '127.0.0.1']
-    ];
-}
-
 return $config;
Index: glados/web/index.php
===================================================================
--- glados.orig/web/index.php
+++ glados/web/index.php
@@ -4,8 +4,8 @@
 setlocale(LC_CTYPE, "C.UTF-8") || setlocale(LC_CTYPE, "en_US.UTF-8");
 
 // comment out the following two lines when deployed to production
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
+defined('YII_DEBUG') or define('YII_DEBUG', false);
+defined('YII_ENV') or define('YII_ENV', 'prod');
 
 require('/usr/share/yii2/autoload.php');
 require('/usr/share/yii2/yiisoft/yii2/Yii.php');
