Remove dev environment
Index: glados/config/web.php
===================================================================
--- glados.orig/config/web.php	2021-03-30 15:38:32.830551227 +0200
+++ glados/config/web.php	2021-03-30 15:40:11.650723297 +0200
@@ -139,21 +139,4 @@
     'runtimePath' => '/var/lib/glados/runtime',
 ];
 
-if (YII_ENV_DEV) {
-    // configuration adjustments for 'dev' environment
-    if(array_key_exists('PATH_INFO', $_SERVER) && $_SERVER['PATH_INFO'] != '/event/stream') {
-        $config['bootstrap'][] = 'debug';
-        $config['modules']['debug'] = [
-            'class' => 'yii\debug\Module',
-            'allowedIPs' => ['10.16.0.222', '192.168.0.248', '127.0.0.1']
-        ];
-    }
-
-    $config['bootstrap'][] = 'gii';
-    $config['modules']['gii'] = [
-        'class' => 'yii\gii\Module',
-        'allowedIPs' => ['10.16.0.222', '192.168.0.248', '127.0.0.1']
-    ];
-}
-
 return $config;
Index: glados/web/index.php
===================================================================
--- glados.orig/web/index.php	2021-03-30 15:38:32.870551298 +0200
+++ glados/web/index.php	2021-03-30 15:40:23.562743806 +0200
@@ -4,8 +4,8 @@
 setlocale(LC_CTYPE, "C.UTF-8") || setlocale(LC_CTYPE, "en_US.UTF-8");
 
 // comment out the following two lines when deployed to production
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
+defined('YII_DEBUG') or define('YII_DEBUG', false);
+defined('YII_ENV') or define('YII_ENV', 'prod');
 
 require('/usr/share/yii2/autoload.php');
 require('/usr/share/yii2/yiisoft/yii2/Yii.php');
